src/main/scala/gestalt/dotty/DottyToolbox.scala

```diff
@@ -524,8 +524,12 @@ class Toolbox(enclosingPosition: Position)(implicit ctx: Context) extends Tbox {
   }
 
   object Case extends CaseImpl {
+    private def ensureBlock(t:TermTree):TermTree = t match {
+      case unchanged:d.Block => unchanged
+      case other => d.Block(Nil,other)
+    }
```

Comment:
In which cases is this method necessary? Currently, its usages seem to be commented out.